rules/code-quality/src/Rector/Ternary/ArrayKeyExistsTernaryThenValueToCoalescingRector.php

```diff
@@ -76,6 +77,10 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
+        if ($this->getValue($node->else) !== 'null' || $node->else instanceof String_) {
```

Comment:
Btw, you can use helper method `$this->isNull($node->else)` here.

Reply:
I tried `$this->isNull`, that not work for this use case.

Reply:
You mean like this?

```php
if (! $this->isNull($node->else)) {
    return null;
}
```

Reply:
thank you, implemented.